libraries/protocol/address.cpp

```diff
@@ -89,12 +89,12 @@ namespace graphene { namespace protocol {
 
    address::operator std::string()const
    {
-        std::array<char,24> bin_addr;
-        static_assert( bin_addr.size() >= sizeof(addr) + 4, "address size mismatch" );
-        memcpy( bin_addr.data(), addr.data(), sizeof(addr) );
+        char bin_addr[24];
```

Comment:
Just a note, the variable is still uninitialized, although it doesn't matter since the data will be overwritten later.

Reply:
Yes, that was intentional.